SU-SOE/grunt-drupal-tasks
# Drupal 7.80 upgrade - SA-CORE-2021-002 security release #32

# READY FOR REVIEW

# Summary
-  Update Drupal 7.78 -> 7.80 per [security release](https://www.drupal.org/sa-core-2021-002). 

# Review By (Date)
- April 26

# Criticality
- 7/10

# Steps to Test

1. Git pull
2. Compile site with Grunt (See documentation)
3. Enable database blog, if it's not already enabled. 
4. Clear the recent logs and run `grunt behat` if set up
5. Check logs for issues.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-499
- https://www.drupal.org/sa-core-2021-002
